operators/tokenizer/bert_tokenizer.cc

```diff
@@ -70,12 +73,43 @@ std::vector<ustring> WordpieceTokenizer::Tokenize(const ustring& text) {
   return result;
 }
 
-std::vector<ustring> WordpieceTokenizer::Tokenize(const std::vector<ustring>& tokens) {
+std::vector<ustring> WordpieceTokenizer::Tokenize(const std::vector<ustring>& tokens, std::list<OffsetMappingType>& offset_map) {
   std::vector<ustring> result;
   for (const auto& token : tokens) {
     GreedySearch(token, result);
   }
 
+  size_t offset = 0;
```

Comment:
Should the following code run only with offset_mapping ouptut being not optional?

Reply:
Yep, that's a great idea, thanks! Will create a new PR for this and do it for CLIP and Roberta as well.